pages/deployment/wifi.md

```diff
@@ -43,35 +97,17 @@ interface, this can be achieved by manually editing the `config.json` file on yo
 
 In general the network config follows the [ConnMan][connman] configuration file format, so you can configure your network in anyway connMan allows. Follow the [official guide][connman-format] for details of how to configure your network if you have more complicated requirements than the our standard configuration.
 
-## Raspberry Pi
-
-The [Raspberry Pi][rpi] can be expanded to connect to a WiFi network by
-installing an adapter:-
+## 3G or Cellular Connections
 
-### Known Working Devices
+Currently 3G or Cellular modems are not supported out of the box, but they are easily setup on a resin.io device, but you will need an ethernet or wifi connection during the setup.
```

Comment:
whoops!! @lifeeth at some point we will should probably migrate some of the knowledge from you blog post directly here and also add a "known working" dongles section in the hardware section of the docs.